tests/testthat/test-global.R

```diff
@@ -60,7 +60,7 @@ test_that("Theme adding works as intended", {
 
 test_that("rename works as intended", {
   expect_identical(
-    rename(list(a=1, b=3, c=4, d=1), c(z="b",y="a",w="f")),
+    rename(list(a=1, b=3, c=4, d=1), b = "z", a = "y", f = "w"),
     list(y=1, z=3, c=4, d=1)
```

Comment:
Slight change in the test, for the reasons mentioned earlier.

```suggestion
    rename2(list(a=1, b=3, c=4, a=2), b="z", f="w", a="y"),
    list(y = 1, z = 3, c = 4, y = 2)
```